syft/frameworks/torch/tensors/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .log import LogTensor  # noqa: F401
```

Comment:
We should make these absolute imports (I think we can just delete each of the .)

Reply:
Will fix